### DISCUSSION TITLE
contentDispositionType cannot found in 'ImageConfigComplete'

### DISCUSSION THREAD

**Discussion by synasapmob** [vercel/next.js#47233] [Help] [answered]
### Summary

in Pull request #46254, config contentDispositionType has been updated, 
but when I implement that had to occur warning at the "terminal", afterward, 
I tried to find the type of ImageConfigComplete and discovered not have any word related to 'contentDispositionType' 

**terminal**
> - The value at .images has an unexpected property, contentDispositionType, which is not in the list of allowed properties (remotePatterns, unoptimized, contentSecurityPolicy, dangerouslyAllowSVG, deviceSizes, disableStaticImages, domains, formats, imageSizes, loader, loaderFile, minimumCacheTTL, path).

**Photo**
> ![Screen Shot 2023-03-17 at 14 44 35](https://user-images.githubusercontent.com/42969529/225843303-87ae423e-e655-474c-b4b2-00ef875a22fd.png)

**Comment by synasapmob** [comment]
@styfle

**Comment by styfle** [comment, marked as answer]
@synasapmob You need to use `next@13.2.0` or newer.

Try running `npm install next@latest` to upgrade.

https://nextjs.org/docs/api-reference/next/image

<img width="510" alt="image" src="https://user-images.githubusercontent.com/229881/225943532-f29e2eea-30c0-4e29-97bc-67b5c72a70a3.png">

**Comment by synasapmob** [comment]
thanks so much
